x/examples/website-wrapper-app/.scripts/build_mobileproxy.sh

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 PLATFORM="$1"
-TAG="${2:-"x/v0.0.1"}"
+TAG="${2:-"x/v0.0.3"}"
```

Comment:
There's a better way. Create a tools.go file instead, like this:
https://github.com/Jigsaw-Code/outline-sdk/blob/main/x/mobileproxy/tools.go

Then go mod tidy will create the go.sum for you